data/json/recipes/recipe_deconstruction.json

```diff
@@ -81,81 +81,113 @@
     ]
   },{
     "type" : "recipe",
-    "result": "creepy_doll",
-    "id_suffix": "uncraft",
-    "category": "CC_NONCRAFT",
-    "subcategory": "CSC_NONCRAFT",
-    "difficulty": 0,
-    "time": 500,
-    "reversible": true,
-    "autolearn": false,
+    "result" : "creepy_doll",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 500,
+    "reversible" : true,
+    "autolearn" : false,
     "qualities" : [ {"id":"SCREW","level":1,"amount":1} ],
-    "components": [
+    "components" : [
       [ [ "scrap", 1 ] ],
       [ [ "plastic_chunk", 2 ] ],
       [ [ "rag", 2 ] ],
       [ [ "RAM", 1 ] ],
       [ [ "e_scrap", 2 ] ]
     ]
-  }, {
+  },{
     "type" : "recipe",
-    "result": "bowl_pewter",
-    "id_suffix": "uncraft",
-    "category": "CC_NONCRAFT",
-    "subcategory": "CSC_NONCRAFT",
-    "difficulty": 0,
-    "time": 500,
-    "reversible": true,
-    "autolearn": false,
-    "qualities":[
-      {"id":"HAMMER","level":1,"amount":1}
+    "result" : "bowl_pewter",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 500,
+    "reversible" : true,
+    "autolearn" : false,
+    "qualities" : [ {"id":"HAMMER","level":1,"amount":1} ],
+    "components" : [
+      [ [ "tin", 100 ] ]
+    ]
+  },{
+    "type" : "recipe",
+    "result" : "cu_pipe",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 1200,
+    "reversible" : true,
+    "autolearn" : false,
+    "qualities" : [ {"id":"HAMMER","level":1,"amount":1} ],
+    "tools": [
```

Comment:
Try to keep all of the alternatives on one line if possible
Each group of requirements however should start a new line